channels/jaicp/src/main/kotlin/com/justai/jaicf/channel/jaicp/dto/config/AsrConfig.kt

```diff
@@ -0,0 +1,127 @@
+package com.justai.jaicf.channel.jaicp.dto.config
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+/**
+ * Configuration parameters for the automatic speech recognition (ASR) provider used in a telephone channel.
+ *
+ * The properties for each ASR provider are different and are defined in the relevant classes.
+ * Only one provider can be specified in the configuration.
+ *
+ * @param type is asr provider of the current session.
+ * @param yandex configuration options for the Yandex ASR provider, if used.
+ * @param zitech configuration options for the Zitech ASR provider, if used.
+ * @param google configuration options for the Google ASR provider, if used.
+ * @param amiVoice configuration options for the AmiVoice ASR provider, if used.
+ * @param mts configuration options for the MTS ASR provider, if used.
+ * @param azure configuration options for the Azure ASR provider, if used.
+ * @param asm configuration options for the ASM ASR provider, if used.
+ * @param sber configuration options for the Sber ASR provider, if used.
+ **/
+@Serializable
+data class AsrConfig(
+    val type: AsrProviderType? = null,
+    val yandex: AsrYandexConfig? = null,
+    val zitech: AsrZitechConfig? = null,
+    val google: AsrGoogleConfig? = null,
+    val amiVoice: AsrAmiVoiceConfig? = null,
```

Comment:
Is it suppossed to be `amiVoice` and not `aimyvoice`?

Reply:
in chatadapter as `amiVoice` variable for asr

Reply:
Maybe we should rename it to `aimyvoice` in code and use `amiVoice` in [SerialName](https://kotlinlang.org/api/kotlinx.serialization/kotlinx-serialization-core/kotlinx.serialization/-serial-name/)?

Reply:
Ok, I'll fix that.

Reply:
Thanks! LGTM now